src/app/ApplicationTemplate.UWP/Views/Styles/Controls/CommandBar.xaml

```diff
@@ -117,8 +118,10 @@
 				Value="{ThemeResource MaterialOnBackgroundBrush}" />
```

Comment:
I see that we have `BasedOn="{StaticResource MaterialCommandBarStyle}"`. Could this be why we're seeing issues? What if we use the native command bar instead?

Reply:
Ditching the material command bar was my first instinct, but isn't it the whole point of material? To not have custom style for that kind of stuff?

Reply:
I have another branch with the command bar style from TZ (with minor modifications) and and it works on all platforms if we want to ditch the material commandbar style.